dbw/node_fw/mod/brake/brake.c

```diff
@@ -101,32 +81,17 @@ static void brake_100Hz()
 {
     static float32_t prev_cmd;
 
-    if (base_dbw_active() && !can_Vel_Cmd_cfg.recieved) {
-        static uint64_t prv_delta_ms;
-        static bool     set;
-
-        if (set) {
-            if (can_Vel_Cmd_cfg.delta_ms - prv_delta_ms >= CMD_TIMEOUT_MS)
-                base_set_state_estop(CAN_DBW_ESTOP_reason_TIMEOUT_CHOICE);
-        } else {
-            prv_delta_ms = can_Vel_Cmd_cfg.delta_ms;
-            set = true;
-        }
-    }
-
-    if (
-        base_dbw_active() &&
-        can_Vel_Cmd_cfg.recieved &&
-        (can_Vel_Cmd_cfg.delta_ms >= CMD_TIMEOUT_MS)
-    )
-        base_set_state_estop(CAN_DBW_ESTOP_reason_TIMEOUT_CHOICE);
+    /* if (!CAN_is_node_DBW_ok()) {
```

Comment:
we have this check in base and read the active status with `base_dbw_active()`, so no need to check this here